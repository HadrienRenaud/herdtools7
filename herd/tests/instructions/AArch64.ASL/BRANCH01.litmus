AArch64 BRANCH01
{

}

 P0           ;
B L0          ;
ADD W0,W0,#1  ;
B .+8         ;
L0:           ;
B .-4         ;
ADD W0,W0,#1  ;
forall 0:X0=1
