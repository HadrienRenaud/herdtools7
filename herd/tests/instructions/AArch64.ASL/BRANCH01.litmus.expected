Test BRANCH01 Required
States 1
0:X0=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=1)
Observation BRANCH01 Always 1 0
Hash=ec1043de14b74d54bab701a4832cb3fa

